config.yaml

```diff
@@ -45,6 +45,17 @@ default:
       gui: false
       memory: 1024
       description: "%{node_name_prefix} #{Time.now.getutc.to_i}"
+    # - _id: "1"
```

Comment:
lets put some comment about how to get this working:
```
# you need to install vagrant-vmware-desktop first before use this provider
# see the installation steps at https://www.vagrantup.com/docs/vmware/installation.html
```

Reply:
updated!